generators/example/src/main/scala/ArianeConfigs.scala

```diff
@@ -0,0 +1,29 @@
+package example
+
+import chisel3._
+
+import freechips.rocketchip.config.{Config}
+
+// ---------------------
+// Ariane Configs
+// ---------------------
+
+class ArianeConfig extends Config(
+  new WithTSI ++                                           // use testchipip serial offchip link
+  new WithNoGPIO ++
+  new WithBootROM ++
+  new WithUART ++
+  new freechips.rocketchip.subsystem.WithNoMMIOPort ++     // no top-level MMIO master port (overrides default set in rocketchip)
+  new freechips.rocketchip.subsystem.WithNoSlavePort ++    // no top-level MMIO slave port (overrides default set in rocketchip)
+  new ariane.WithNArianeCores(1) ++                        // single Ariane core
+  new freechips.rocketchip.system.BaseConfig)              // "base" rocketchip system
+
+class dmiArianeConfig extends Config(
```

Comment:
What is the purpose of this config?

Reply:
In the base Ariane repo, they use the DMI Debug bringup method. This was used to match that.

Reply:
Why do we have to match that? I'd prefer if there is just one Ariane config.

Reply:
While I agree, having one config would be fine... we have to "hack" the "main config" (ie `ArianeConfig`) so that the `tohost/fromhost` addresses are not cached. So until that is truely fixed (with an L1.5 adapter to TL) I would like to keep the other config as backup.